spec/acceptance/utf8_encoding_spec.rb

```diff
@@ -15,8 +16,15 @@ class { 'postgresql::server': }
 
   it 'with defaults' do
     export_locales('en_NG.UTF8')
-    idempotent_apply(pp)
-    expect(port(5432)).to be_listening
+    idempotent_apply(pp, debug: true)
+    puts '-------------------------------'
+    puts LitmusHelper.instance.run_shell('ss -lntp').stdout
+    puts '-------------------------------'
+    puts LitmusHelper.instance.run_shell('journalctl -u postgresql').stdout
+    puts '-------------------------------'
+    puts LitmusHelper.instance.run_shell('systemctl status postgresql*').stdout
+    puts '-------------------------------'
+    expect(port(5432)).to be_listening.on('127.0.0.1').with('tcp')
```

Comment:
the problem I had in the past: some systems report tcp, some tcp4. that was really ugly so I switched to matching the IP address. and that sometimes didn't work because some CI systems have IPv6 so it binds to ::1 but sometimes it's only 127.0.0.1. all ugly...

Reply:
I am doing more test because this is quite weird.  Looking at the code it *should* work, looking at `ss` output it *seems* to be working, but it is not and that makes me feel angry.  Similar code work in other areas of the project so I guess something is very wrong…  I would like to nail it rather than ignoring it if possible.